packages/functional-tests/tests/oauth/syncSignIn.spec.ts

```diff
@@ -105,9 +125,13 @@ test.describe('severity-1 #smoke', () => {
       if (config.showReactApp.signUpRoutes !== true) {
         await login.fillOutFirstSignUp(email, password);
       } else {
-        await signupReact.fillOutEmailFirst(email);
-        await signupReact.fillOutSignupForm(password);
-        await signupReact.fillOutCodeForm(email);
+        await signupReact.fillOutEmailForm({ email: email, submit: true });
+        await signupReact.fillOutSignupForm({
+          password: password,
+          age: AGE_21,
```

Comment:
Similar to another comment, I wonder if we could simplify this to just `await signupReact.fillOutSignupForm({ password })`. `age` could get a default value of that constant and `submit` could have a default of `true`.

Reply:
In the case of `age` I'd actually like to keep it explicit.

I have a strong preference for exposing form data at the top level of a test. The reasoning behind this practice is to facilitate manual execution of test steps. The top level test method acts like a set of instructions and we aren't forced to step into configurations, utilities or page objects to find input values. The downside is verbosity.